File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean answer = false;
        Arrays.sort(nums);

        for(int i = 1; i < nums.length; i++){
            if(nums[i] == nums[i - 1]){
                answer = true;
                return answer;
            }
        }
        return answer;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashmap = dict()

        for num in nums:
            if num in hashmap:
                hashmap[num] += 1
            else:
                hashmap[num] = 1

        freq_list = []
        for key in hashmap:
            freq_list.append(hashmap[key])

        freq_list.sort()
        result = []
        kth_freq = freq_list[len(freq_list) - k]
        for key in hashmap:
            if hashmap[key] >= kth_freq:
                result.append(key)

        return result

        
